debugging/README.md

```diff
@@ -1,6 +1,6 @@
 ## Debugging API
```

Comment:
Thanks for updating this document!  

I am in favor of describing only the latest debugging options in the top section as you have shown in your `Node >= 8` section. Then move the legacy options into a new section below.

For example:
## Debugging API
Starting with Node 8 ...

## Legacy Debugging APIs
Prior to Node 8 ...